src/Sulu/Bundle/WebsiteBundle/Twig/Navigation/NavigationTwigExtension.php

```diff
@@ -210,7 +210,12 @@ private function getUser(): ?UserInterface
             return null;
         }
 
-        $user = $this->tokenStorage->getToken()->getUser();
+        $token = $this->tokenStorage->getToken();
```

Comment:
If I remember correctly this is not the only code part where the token is read and the user accessed. Can you check for the other `$this->tokenStorage->getToken()` calls and fix it also there.

Reply:
there are severall places where we do that :) but in the most its correct (some are now fixed) - only in some admin services i dont think its neccessary to change it - there is the user required and it seems that this "edge case" cannot happen there.